symbulate/distributions.py

```diff
@@ -556,6 +585,87 @@ def draw(self):
 
         return np.random.f(self.dfN, self.dfD)
 
+class Cauchy(Distribution):
+    """Defines a probability space for a Cauchy distribution
+
+    Attributes:
+      The Cauchy distribution has no parameters  
+    """
+
+    def __init__(self):
+        params = {}
+        super().__init__(params, stats.cauchy, False)
+
+    def draw(self):
+        return np.random.standard_cauchy()
+
+class LogNormal(Distribution):
+    """Defines a probability space for a Log-Normal distribution
+
+    Attributes:
+      mean (float): mean of log-normal distribution 
+      var (float): variance of log-normal distribution
+      sd (float): standard deviation of log-normal distribution
+        (if specified, var parameter will be ignored)
+    """
+
+    def __init__(self, mean=0.0, var=1.0, sd=None):
+        if sd is None:
+            self.s = np.sqrt(var)
+        else:
+            self.s = sd
+
+        params = {
+            "s" : self.s,
+            "scale" : np.exp(mean) 
+            }
+        super().__init__(params, stats.lognorm, False)
+        self.xlim = (0, self.xlim[1]) # Log-Normal distributions are not defined for x < 0
+
+    def draw(self):
+        return np.random.lognormal(self.mean(), self.sd())
```

Comment:
This implementation isn't right. self.mean() and self.sd() are the EV and SD of the LogNormal distribution. They are NOT the original mean and sd parameters. If you want to use those, you'll have to store those values. But you can't call them .mean and .sd, since those are reserved.